src/api/ppx_deriving.cppo.ml

```diff
@@ -507,11 +561,13 @@ let fold_exprs ?unit fn exprs =
     | None -> raise (Invalid_argument "Ppx_deriving.fold_exprs")
 
 let seq_reduce ?sep a b =
+  let loc = !Ast_helper.default_loc in
```

Comment:
I wondered whether we can give a better location to this (join the locations of `a` and `b`). In fact `default_loc` is already a reasonable choice, as the ppx_deriving mapper already calls `Ast_helper.with_default_loc` to set the default location when generating code from a declaration -- the location points to the whole declaration that is currently being processed. Good enough.